### DISCUSSION TITLE
Command line - output report to csv ?

### DISCUSSION THREAD

**Discussion by joeyfall** [WinMerge/winmerge#1341] [Q&A]
Hi all,

Is it possible to use the command line with the /or option to produce an output report in  .csv format ?

By default the output appears to be created as a html file, and specifying csv in the output file name just creates a html file with a .csv extension.

For my use case, I'm comparing two folders using size only.
Both of these command lines give output that is a html file 

"C:\Program Files\WinMerge\winmergeu.exe"  /m Size \\source_parent\folder \\target_parent\folder /or **_g:\folder_comp_report.html_**

"C:\Program Files\WinMerge\winmergeu.exe"  /m Size \\source_parent\folder \\target_parent\folder /or **_g:\folder_comp_report.csv_**

Thanks for looking !
<img width="390" alt="csv_still_html" src="https://user-images.githubusercontent.com/105742178/168875448-fd75aeda-42d4-4aa0-8852-01173f4c408b.png">

**Comment by sdottaka** [comment]
If it is a folder comparison report, you can output it as a CSV file by adding the following options.

`/cfg ReportFiles/ReportType=0`

Unfortunately, there is no option to output the file comparison report in CSV format.

However, since the output HTML file can be opened in Excel, it is possible to make it a CSV file.
